capsules/src/acifc.rs

```diff
@@ -0,0 +1,61 @@
+//! Provides userspace access to the ACIFC interface.
```

Comment:
`ACIFC` is a pretty SAM4L-specific name. For platform-agnostic components, like the system-call capsule, we should use names that are more meaningful from an application perspective. Perhaps: "AnalogComparator"

Reply:
I've thought about this for a bit, but reckoned "AC/ACs" was too short to be clear and "AnalogComparators" was too long and might make the code unnecessarily large. How about I change the names at the top to "Analog Comparators", and keep the use of "Acifc" in the code the same?